cmake/Tools/Platform/Android/android_deployment.py

```diff
@@ -183,7 +183,7 @@ def adb_call(self, arg_list, device_id=None):
         logging.debug(f"adb command: {subprocess.list2cmdline(call_arguments)}")
 
         try:
-            output = subprocess.check_output(call_arguments,
+            output = subprocess.check_output(subprocess.list2cmdline(call_arguments),
```

Comment:
Whoops, good catch. You can't use shell=True with separate argument array as an input.